docs/changes.rst

```diff
@@ -2,6 +2,10 @@
 
 Changelog
 ---------
```

Comment:
need to pull updates from master which is on 1.3.9 and update changelog with everything in this PR!

Reply:
bump to 1.4.0